omod/src/main/java/org/openmrs/module/patientdashboard/web/controller/global/LabOrderListItem.java

```diff
@@ -53,12 +53,12 @@ public LabOrderListItem(Order order) {
 		patientId = order.getPatient().getPatientId();
 		conceptId = order.getConcept().getConceptId();
 		conceptShortName = order.getConcept().getName().getShortName().isEmpty() ? order.getConcept().getName().getName() : order.getConcept().getName().getShortName();
-		startDateString = order.getStartDate() == null ? null : Context.getDateFormat().format(order.getStartDate());
+		startDateString = order.getDateActivated() == null ? null : Context.getDateFormat().format(order.getDateActivated());
 		orderDateString = order.getDateCreated() == null ? null : Context.getDateFormat().format(order.getDateCreated());
 		encounterDateString = order.getEncounter().getEncounterDatetime() == null ? null : Context.getDateFormat().format(order.getEncounter().getEncounterDatetime());
 		locationId = order.getEncounter().getLocation().getLocationId();
 		locationName = order.getEncounter().getLocation().getName();
-		discontinuedDateString = order.getDiscontinuedDate() == null ? null : Context.getDateFormat().format(order.getDiscontinuedDate());
+		discontinuedDateString = order.getDateStopped() == null ? null : Context.getDateFormat().format(order.getDateStopped());

```

Comment:
For this code section looks like you have changed the contents to reflect what the current OpenMRS version is, that is great! but there are other sections that are depended on 1.9.8 and how they are put in the data model. Just maintain the version and through their maven dependencies, they will pull the required artifacts in your local machine. The main goal with these modules is to make them run on kenyaEMR with minimal changes. Have you tried these on your local environment?

Reply:
thanks. I tried it and it works fine

Reply:
call files that are being provided by 1.9.8 for now, we will upgrade later

Reply:
This is noted